<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVMInferenceConfig</class>
 <widget class="QWidget" name="SVMInferenceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM Inference Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="svmGroupBox">
     <property name="title">
      <string>SVM inference</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="ubHandGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Connect to UBHand</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label7">
           <property name="text">
            <string>JSON with gesture mapping:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="browseJSONButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label8">
           <property name="text">
            <string>Path to JSON:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="mappingJSONPathLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>Signal:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
          <item>
           <widget class="QComboBox" name="signalComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="rescanSignalsButton">
            <property name="toolTip">
             <string>Rescan signals</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="view-refresh">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label2">
          <property name="text">
           <string>Feature selection:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="featureComboBox">
          <item>
           <property name="text">
            <string>Waveform length</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RMS</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label3">
          <property name="text">
           <string>Window size (ms):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="winSizeTextField">
          <property name="toolTip">
           <string>If a non-numeric value is set, the default value will be used</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label4">
          <property name="text">
           <string>SVM model:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="browseModelButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label5">
          <property name="text">
           <string>Path to SVM model:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="svmModelPathLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label6">
          <property name="text">
           <string>Predicted label:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="svmPredLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
